scala/lambdas/ingest-asset-reconciler/src/main/scala/uk/gov/nationalarchives/ingestassetreconciler/Lambda.scala

```diff
@@ -64,6 +65,15 @@ class Lambda extends LambdaRunner[Input, StateOutput, Config, Dependencies] {
       titleOfCoWithoutExtension == assetChildTitleOrFileNameWithoutExtension
     }
 
+  private def doesChecksumMatchFixity(item: DynamoFormatters.FileDynamoTable, fixities: List[Client.Fixity]): Boolean = {
+    val sortedItems = item.checksums.sortBy(_.algorithm)
```

Comment:
Maybe `sortedChecksums`? `sortedItems` makes it sound like the DynamoDB row is sorted